README.md

```diff
@@ -65,6 +69,17 @@ and [**push-to-registry**](https://github.com/redhat-actions/push-to-registry) a
 
 For a complete example see the [example workflow](.github/workflows/example.yml).
 
+<a id="using-private-images"></a>
+
+## Using private images
```

Comment:
Are you talking about this?

Reply:
uh. Yes I am. sorry didn't read close enough

Reply:
I referred this in input `create_pull_secret_from`